docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -171,36 +171,40 @@ This allows us to rely on [Hooks](https://reactjs.org/docs/hooks-intro.html).
 
 ### Table
 
-- Remove the deprecated `numeric` property.
+- [TableCell] Remove the deprecated `numeric` property.
 
   ```diff
   -<TableCell numeric>{row.calories}</TableCell>
   +<TableCell align="right">{row.calories}</TableCell>
   ```
-- Remove the fixed height CSS property on the `TableRow`.
+- [TableRow] Remove the fixed height CSS property.
   The cell height is computed by the browser using the padding and line-height.
-- Move the `dense` mode to a different property:
+- [TableCell] Move the `dense` mode to a different property:
 
   ```diff
   -<TableCell padding="dense" />
   +<TableCell size="small" />
   ```
-
-- The `TablePagination` component no longer tries to fix invalid (`page`, `count`, `rowsPerPage`) property combinations. It raises a warning instead.
+- [TablePagination] The component no longer tries to fix invalid (`page`, `count`, `rowsPerPage`) property combinations. It raises a warning instead.
 
 ### Tabs
 
-- Remove the `labelContainer`, `label` and `labelWrapped` class keys for simplicity.
+- [Tab] Remove the `labelContainer`, `label` and `labelWrapped` class keys for simplicity.
 This has allowed us to removed 2 intermediary DOM elements.
 You should be able to move the custom styles to the root class key.
 
   ![capture d ecran 2019-02-23 a 15 46 48](https://user-images.githubusercontent.com/3165635/53287870-53a35500-3782-11e9-9431-2d1a14a41be0.png)
 
 ### Menu
 
-- Remove the fixed height of the MenuItem.
+- [MenuItem] Remove the fixed height of the MenuItem.
   The padding and line-height are used by the browser to compute the height.
 
+### Dialog
+
+- [DialogActions] `action` CSS class is applied to root element if `disableActionSpacing={false}` instead of children.
```

Comment:
[DialogActions] `action` CSS class is applied to root element instead of children if `disableActionSpacing={false}`. ?

Reply:
Looks like it https://github.com/mui-org/material-ui/pull/14795/files#diff-11834bafde7456461f207ce0759a33b0L28

Could you format this as a suggestion so that I can batch it with the other one? To lazy to go back to the branch ;)

Reply:
Done :)

Reply:
Can't batch suggestions from different commits unfortunately. Better luck next time :smile: